### ISSUE TITLE
Add features in Voice Assistant 

### ISSUE THREAD

**Issue by Nitesh-thapliyal** [Bauddhik-Geeks/ATOM-The-Voice-Assistant#2]

**Comment by ghost** [comment]
Hi sir , I can try adding a feature"alarm clock" using python , but I am not sure if it will work or not as I had recently learned python and don't have any prior experience in open source contributions , so can I give it a try if you give permission?

**Comment by adityamangal1** [comment]
Yeah @Luckyboy27, you can fork and clone this repo in your local system and test your feature adding to this repo code there

**Comment by adityamangal1** [comment]
Also join the organization and discord server first for any other queries and your contribution is counted as hacktoberfest accepted.😊

Organization link👉 https://github.com/Bauddhik-Geeks/Welcome-to-Bauddhik-Geeks
Discord server link 👉 https://join.bauddhikgeeks.tech

**Comment by ghost** [comment]
> Yeah @Luckyboy27, you can fork and clone this repo in your local system and test your feature adding to this repo code there

Ok sir , thanks for confirming, I will try my best .

**Comment by adityamangal1** [comment]
@Luckyboy27 Also join the organization and discord server first for any other queries and your contribution is counted as hacktoberfest accepted.😊

Organization link👉 https://github.com/Bauddhik-Geeks/Welcome-to-Bauddhik-Geeks
Discord server link 👉 https://join.bauddhikgeeks.tech

**Comment by ghost** [comment]
@adityamangal1 thanks for the invitation sir and now I had  joined sir

**Comment by faisaljamil25** [comment]
@adityamangal1 can I also work on this? I can add features like "search videos on youtube", "create to-do list", "translation" and more.

**Comment by Nitesh-thapliyal** [comment]
Go ahead @faisaljamil25 👍

**Comment by Mrinank-Bhowmick** [comment]
Hey I want to add 'write a note' 📝feature by which user will dictate and a new text file will be created with the note written on it.😊

**Comment by DebasishPH** [comment]
Hello I would like to work on this project, I am thinking of adding the feature to 'set important dates on the calendar', kindly assign me.

**Comment by adwaiyrandale** [comment]
I would like to add a feature that plays a song from spotify, 
kindly assign me

**Comment by Nitesh-thapliyal** [comment]
> I would like to add a feature that plays a song from spotify, kindly assign me

nice 👍
